File: C/PRAK301-2210817110006-MohammadZakiFirmansah.c
#include <stdio.h>
int main(){
    int Angka1, Angka2, Angka3;
    printf("Mengurutkan bilangan dari terkecil ke terbesar\n");
    printf("Angka 1 : ");
    scanf("%d", &Angka1);
    printf("Angka 2 : ");
    scanf("%d", &Angka2);
    printf("Angka 3 : ");
    scanf("%d", &Angka3);
    if(Angka1>Angka2 && Angka1>Angka3 && Angka2>Angka3){
        printf("%d, %d, %d",Angka3, Angka2, Angka1);  
    }else if (Angka2>Angka1 && Angka2>Angka3 && Angka1>Angka3){
        printf("%d, %d, %d",Angka3, Angka1, Angka2);
    }else if (Angka1>Angka2 && Angka1>Angka3 && Angka2<Angka3){
        printf("%d, %d, %d",Angka2, Angka3, Angka1);
    }else if (Angka2>Angka1 && Angka2>Angka3 && Angka1<Angka3){
        printf("%d, %d, %d",Angka1, Angka3, Angka2);
    }else if (Angka3>Angka1 && Angka3>Angka2 && Angka1>Angka2){
        printf("%d, %d, %d",Angka2, Angka1, Angka3);
    }else{
        printf("%d, %d, %d",Angka1, Angka2, Angka3);
    }   
}     

File: Python/PRAK301-2210817110006-MohammadZakiFirmansah.py
print("Mengurutkan bilangan dari terkecil ke terbesar")
a = int(input("Angka 1 : "))
b = int(input("Angka 2 : "))
c = int(input("Angka 3 : "))
if (a>b and a>c and b>c) :
        print(c, b, a)
elif (a>b and a>c and b<c) :
        print(b, c, a)  
elif (b>a and b>c and a>c) :
        print(c, a, b)
elif (b>a and b>c and a<c) :
        print(a, c, b)
elif (c>a and c>b and a>b) :
        print(b, a, c)
else :
        print(a, b, c)